src/librustc_typeck/astconv/mod.rs

```diff
@@ -1,33 +1,31 @@
-// ignore-tidy-filelength FIXME(#67418) Split up this file.
```

Comment:
you don't need to move the file. It can either be `astconv/mod.rs` or `astconv.rs` and it would work